src/js/jwt/jwt.unit.test.js

```diff
@@ -264,7 +264,7 @@ describe('JWT', () => {
         });
       });
 
-      describe('token update fails', () => {
+      xdescribe('token update fails', () => {
```

Comment:
typo?

Reply:
More like forgotten `xdescribe`. Thank you!